dist/ci/obs-ic/api.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+
+su - frontend -c "RAILS_ENV=production rails r \"Project.find_or_create_by(remoteurl: 'https://api.opensuse.org/public', name: 'openSUSE.org').store\""
```

Comment:
I guess I don't even need this by default - and one less .sh file to worry about.